modules/mod_xfer.c

```diff
@@ -2513,6 +2513,15 @@ MODRET xfer_retr(cmd_rec *cmd) {
     }
 
     len = transmit_data(cmd->pool, nbytes_sent, &curr_pos, lbuf, bufsz);
+
+	/*waiting 2 secons to check if the file is growing*/
+	sleep(2);
```

Comment:
Hardcoded `sleep(3)` calls are terrible; they should _at least_ be configurable.  In this particular case, why 2 seconds, rather than 1, or 5, or some other arbitrary number?